index.ts

```diff
@@ -207,10 +207,12 @@ addTests('isProjects', [
 	'https://github.com/sindresorhus/refined-github/projects',
 ]);
 
-export const isDiscussion = (url: URL | HTMLAnchorElement | Location = location): boolean => /^discussions\/\d+/.test(getRepo(url)?.path ?? getOrg(url)?.path!);
+export const isDiscussion = (url: URL | HTMLAnchorElement | Location = location): boolean => /^discussions\/\d+/.test(getRepo(url)?.path ?? getOrg(url)?.path!) || getRepo(url)?.path === 'discussions/new';
```

Comment:
```suggestion
export const isDiscussion = (url: URL | HTMLAnchorElement | Location = location): boolean => /^discussions\/(\d+|new)/.test(getRepo(url)?.path ?? getOrg(url)?.path!);
```
This will be more concise, WDYT?

Reply:
Cool, apply it. see [f3405fc](https://github.com/refined-github/github-url-detection/pull/177/commits/f3405fc1e9db9ec91922f8aaaabd4eda3a1f735c)